src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/get-started.mdx

```diff
@@ -0,0 +1,412 @@
+
+1. Run the `get-login-password` command to authenticate the Docker CLI to your Amazon ECR registry.
+2. Set the `--region` value to the AWS Region where you want to create the Amazon ECR repository.
+3. Replace 111122223333 with your AWS account ID.
+```
+    aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 111122223333.dkr.ecr.us-east-1.amazonaws.com
+```
+4. Create a repository in Amazon ECR using the create-repository command.
+```
+aws ecr create-repository --repository-name hello-world --image-scanning-configuration scanOnPush=true --image-tag-mutability MUTABLE
+```
+5. Copy the `repositoryUri` from the output in the previous step.
+6. Run the [docker tag](https://docs.docker.com/reference/cli/docker/image/tag/) command to tag your local image into your Amazon ECR repository as the latest version. In this command, replace `docker-image:test` with the name and tag of your Docker image. Then replace &lt; ECRrepositoryUri &gt; with the `repositoryUri` you copied. Ensure that you include `:latest` at the end of the URI.
+
+```
+docker tag docker-image:test <ECRrepositoryUri> :latest
+EX: docker tag docker-image:test 111122223333.dkr.ecr.us-east-1.amazonaws.com/hello-world:latest
+```
+7. Now, run the docker push command to deploy your local image to the Amazon ECR repository. Ensure you include `:latest` at the end of the repository URI.
+```
+docker push 111122223333.dkr.ecr.us-east-1.amazonaws.com/hello-world:latest
+```
+</Step>
+<Step>
+## Create a Lambda containerized function [#create-containerized-function]
+
+Now, you're ready to create a Lambda containerized function. 
+
+You'll need to create an execution role for the function if you don't already have one. If you're not sure how to do this, please refer to the [AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-awscli.html#with-userapp-walkthrough-custom-events-create-iam-role). For the next step, you will need the Amazon Resource Name, or ARN, of the role. 
+
+Now, you are ready to create the Lambda function. When completing the following command, you must specify the repository URI from earlier for the `Image` and `ImageUri` fields. Ensure that you include `:latest` at the end of the URI.
+
+```
+aws lambda create-function \ --function-name hello-world \ --package-type Image \ --code ImageUri=111122223333.dkr.ecr.us-east-1.amazonaws.com/hello-world:latest \ --role arn:aws:iam::111122223333:role/lambda-ex
+```
+Next, you have to invoke the function:
+
+```
+aws Lambda invoke --function-name hello-world response.json
+```
+
+If you have any questions, see [AWS's documentation for deploying Lambda functions with container images](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-image.html).
+</Step>
+
+<Step>
+## Add a New Relic Lambda layer as an image [#add-new-relic-image]
```

Comment:
These below steps are already mentioned in enable-containerized-function.mdx file, So not required here.